backend/src/injector.py

```diff
@@ -412,7 +412,7 @@ async def get_tab_lambda(test: Callable[[Tab], bool]) -> Tab:
 
 SHARED_CTX_NAMES = ["SharedJSContext", "Steam Shared Context presented by Valve™", "Steam", "SP"]
 CLOSEABLE_URLS = ["about:blank", "data:text/html,%3Cbody%3E%3C%2Fbody%3E"] # Closing anything other than these *really* likes to crash Steam
-DO_NOT_CLOSE_URLS = ["Valve Steam Gamepad/default", "Valve%20Steam%20Gamepad/default"] # Steam Big Picture Mode tab
+DO_NOT_CLOSE_URLS = ["Valve Steam Gamepad/default", "Valve%20Steam%20Gamepad"] # Steam Big Picture Mode tab
```

Comment:
Seems like we accept multiple of these. Any reason you chose to remove the previous one rather than add a new option? Just a little concerned about backward compatibility.

Reply:
We've asked the question in the discord, it's because `Valve%20Steam%20Gamepad/default` includes `Valve%20Steam%20Gamepad` so it will trigger anyway.